glue/viewers/profile/qt/options_widget.py

```diff
@@ -33,10 +36,24 @@ def __init__(self, viewer_state, session, parent=None):
 
         self.session = session
 
+        for dataset in self.session.data_collection:
+            if isinstance(dataset, IndexedData):
+                self.viewer_state.indices = dataset.indices
```

Comment:
So is the idea that for indexdata for example, the sliders can be used in the image viewer to change the actual indices of the IndexedData?

Reply:
Yes, and `SlicedData` is to supplement `IndexedData` for handling the slicing of data cubes. So in terms of functionality these two are distinct.